editor/SCsub

```diff
@@ -11,15 +11,17 @@ from compat import open_utf8
 import editor_builders
 
 
-def _make_doc_data_class_path(to_path):
+def _make_modules_doc_data(to_path, doc_pathmap):
```

Comment:
It's not only modules though since it includes `doc/classes`?

Reply:
If you look at generated header:

<details>
<summary>
doc_data_class_path.gen.h
</summary>

```cpp
static const int _doc_data_class_path_count = 96;
struct _DocDataClassPath { const char* name; const char* path; };
static const _DocDataClassPath _doc_data_class_paths[97] = {
	{"@GDScript", "modules/gdscript/doc_classes"},
	{"@NativeScript", "modules/gdnative/doc_classes"},
	{"@VisualScript", "modules/visual_script/doc_classes"},
	{"ARVRInterfaceGDNative", "modules/gdnative/doc_classes"},
	{"AudioStreamOGGVorbis", "modules/stb_vorbis/doc_classes"},
	{"AudioStreamOpus", "modules/opus/doc_classes"},
	{"BulletPhysicsDirectBodyState", "modules/bullet/doc_classes"},
	{"BulletPhysicsServer", "modules/bullet/doc_classes"},
	{"CSGBox", "modules/csg/doc_classes"},
	{"CSGCombiner", "modules/csg/doc_classes"},
	{"CSGCylinder", "modules/csg/doc_classes"},
	{"CSGMesh", "modules/csg/doc_classes"},
	{"CSGPolygon", "modules/csg/doc_classes"},
	{"CSGPrimitive", "modules/csg/doc_classes"},
	{"CSGShape", "modules/csg/doc_classes"},
	{"CSGSphere", "modules/csg/doc_classes"},
	{"CSGTorus", "modules/csg/doc_classes"},
	{"GDNative", "modules/gdnative/doc_classes"},
	{"GDNativeLibrary", "modules/gdnative/doc_classes"},
	{"GDScript", "modules/gdscript/doc_classes"},
	{"GDScriptFunctionState", "modules/gdscript/doc_classes"},
	{"GDScriptNativeClass", "modules/gdscript/doc_classes"},
	{"GridMap", "modules/gridmap/doc_classes"},
	{"MobileVRInterface", "modules/mobile_vr/doc_classes"},
	{"MultiplayerPeerGDNative", "modules/gdnative/doc_classes"},
	{"NativeScript", "modules/gdnative/doc_classes"},
	{"NetworkedMultiplayerENet", "modules/enet/doc_classes"},
	{"NoiseTexture", "modules/opensimplex/doc_classes"},
	{"OpenSimplexNoise", "modules/opensimplex/doc_classes"},
	{"PacketPeerGDNative", "modules/gdnative/doc_classes"},
	{"PluginScript", "modules/gdnative/doc_classes"},
	{"RegEx", "modules/regex/doc_classes"},
	{"RegExMatch", "modules/regex/doc_classes"},
	{"StreamPeerGDNative", "modules/gdnative/doc_classes"},
	{"UPNP", "modules/upnp/doc_classes"},
	{"UPNPDevice", "modules/upnp/doc_classes"},
	{"VideoStreamGDNative", "modules/gdnative/doc_classes"},
	{"VideoStreamTheora", "modules/theora/doc_classes"},
	{"VideoStreamWebm", "modules/webm/doc_classes"},
	{"VisualScript", "modules/visual_script/doc_classes"},
	{"VisualScriptBasicTypeConstant", "modules/visual_script/doc_classes"},
	{"VisualScriptBuiltinFunc", "modules/visual_script/doc_classes"},
	{"VisualScriptClassConstant", "modules/visual_script/doc_classes"},
	{"VisualScriptComment", "modules/visual_script/doc_classes"},
	{"VisualScriptComposeArray", "modules/visual_script/doc_classes"},
	{"VisualScriptCondition", "modules/visual_script/doc_classes"},
	{"VisualScriptConstant", "modules/visual_script/doc_classes"},
	{"VisualScriptConstructor", "modules/visual_script/doc_classes"},
	{"VisualScriptCustomNode", "modules/visual_script/doc_classes"},
	{"VisualScriptDeconstruct", "modules/visual_script/doc_classes"},
	{"VisualScriptEditor", "modules/visual_script/doc_classes"},
	{"VisualScriptEmitSignal", "modules/visual_script/doc_classes"},
	{"VisualScriptEngineSingleton", "modules/visual_script/doc_classes"},
	{"VisualScriptExpression", "modules/visual_script/doc_classes"},
	{"VisualScriptFunction", "modules/visual_script/doc_classes"},
	{"VisualScriptFunctionCall", "modules/visual_script/doc_classes"},
	{"VisualScriptFunctionState", "modules/visual_script/doc_classes"},
	{"VisualScriptGlobalConstant", "modules/visual_script/doc_classes"},
	{"VisualScriptIndexGet", "modules/visual_script/doc_classes"},
	{"VisualScriptIndexSet", "modules/visual_script/doc_classes"},
	{"VisualScriptInputAction", "modules/visual_script/doc_classes"},
	{"VisualScriptIterator", "modules/visual_script/doc_classes"},
	{"VisualScriptLists", "modules/visual_script/doc_classes"},
	{"VisualScriptLocalVar", "modules/visual_script/doc_classes"},
	{"VisualScriptLocalVarSet", "modules/visual_script/doc_classes"},
	{"VisualScriptMathConstant", "modules/visual_script/doc_classes"},
	{"VisualScriptNode", "modules/visual_script/doc_classes"},
	{"VisualScriptOperator", "modules/visual_script/doc_classes"},
	{"VisualScriptPreload", "modules/visual_script/doc_classes"},
	{"VisualScriptPropertyGet", "modules/visual_script/doc_classes"},
	{"VisualScriptPropertySet", "modules/visual_script/doc_classes"},
	{"VisualScriptResourcePath", "modules/visual_script/doc_classes"},
	{"VisualScriptReturn", "modules/visual_script/doc_classes"},
	{"VisualScriptSceneNode", "modules/visual_script/doc_classes"},
	{"VisualScriptSceneTree", "modules/visual_script/doc_classes"},
	{"VisualScriptSelect", "modules/visual_script/doc_classes"},
	{"VisualScriptSelf", "modules/visual_script/doc_classes"},
	{"VisualScriptSequence", "modules/visual_script/doc_classes"},
	{"VisualScriptSubCall", "modules/visual_script/doc_classes"},
	{"VisualScriptSwitch", "modules/visual_script/doc_classes"},
	{"VisualScriptTypeCast", "modules/visual_script/doc_classes"},
	{"VisualScriptVariableGet", "modules/visual_script/doc_classes"},
	{"VisualScriptVariableSet", "modules/visual_script/doc_classes"},
	{"VisualScriptWhile", "modules/visual_script/doc_classes"},
	{"VisualScriptYield", "modules/visual_script/doc_classes"},
	{"VisualScriptYieldSignal", "modules/visual_script/doc_classes"},
	{"WebRTCDataChannel", "modules/webrtc/doc_classes"},
	{"WebRTCDataChannelGDNative", "modules/gdnative/doc_classes"},
	{"WebRTCMultiplayer", "modules/webrtc/doc_classes"},
	{"WebRTCPeerConnection", "modules/webrtc/doc_classes"},
	{"WebRTCPeerConnectionGDNative", "modules/gdnative/doc_classes"},
	{"WebSocketClient", "modules/websocket/doc_classes"},
	{"WebSocketMultiplayerPeer", "modules/websocket/doc_classes"},
	{"WebSocketPeer", "modules/websocket/doc_classes"},
	{"WebSocketServer", "modules/websocket/doc_classes"},
	{NULL, NULL}
};
```
</details>

You see how those include module doc paths only. Looks like it's used for doc dump for classes with non-default documentation path. I'm not sure whether it makes sense to generalize the usage of this header to include other non-core stuff.

So the process of generating *doc data* and *doc paths* are different here. `_make_modules_doc_data` could be renamed further to `_make_modules_doc_paths` to be more specific.